server/views/pages/activities/record-attendance/not-attended-reason.njk

```diff
@@ -149,7 +149,7 @@
 
                 }
             ]
-        }) }}
+        }) if isPayable }}
```

Comment:
This isn't in the ACs, but I'm removing this radio selection if the activity isn't payable since we currently only use this input for deciding whether an "OTHER" attendance reason is to be paid (which an unpaid activity can never be).

When Dave is back in I'll confirm whether we need a way to check if an "OTHER" attendance is marked acceptable or unacceptable.